src/Page/Food/Builder.elm

```diff
@@ -773,11 +804,36 @@ packagingListView db selectedImpact recipe results =
     ]
 
 
+transportToDistributionView : Impact.Definition -> Recipe -> Recipe.Results -> List (Html Msg)
+transportToDistributionView selectedImpact recipe results =
+    [ span [ class "w-50" ] [ text "" ]
+    , div [ class "DownArrow" ]
```

Comment:
Note: 
- it's a little strange we have an arrow here but not for the other steps
- also ideally we'd update other down arrows to use this new style (maybe in a later patch)

Reply:
Yeah, we only need some information here, but we could still have arrows between each step, I'll add this.

I wasn't sure you wanted me to replace the textile arrows, but I can ;)